src/plugins/mer/meremulatormodedetailswidget.ui

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Mer::Internal::MerEmulatorModeDetailsWidget</class>
+ <widget class="QWidget" name="Mer::Internal::MerEmulatorModeDetailsWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>824</width>
+    <height>463</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout_2">
+   <item>
+    <layout class="QHBoxLayout" name="horizontalLayout">
+     <item>
+      <widget class="QLabel" name="label_2">
+       <property name="text">
+        <string>Name:</string>
+       </property>
+       <property name="buddy">
+        <cstring>nameLineEdit</cstring>
```

Comment:
Under the "Name" field, I would add also the information if the model is sdk-provided or user-created. I would do it similarly how it is done for devices, so the field name would be "Auto-detected". When adding this, I would add group box "General" for these two, like it is done for the devices.

Reply:
Done.